code/modules/clothing/head/misc_special.dm

```diff
@@ -213,3 +214,30 @@
 	icon_state = "foilhat"
 	item_state = "paper"
 	siemens_coefficient = 2
+
+/obj/item/clothing/head/celtic
+	name = "\improper Celtic crown"
+	desc = "According to legend, Celtic kings would use crowns like this one to shield their subjects from harsh winters back on Earth."
+	icon_state = "celtic_crown"
+	wizard_garb = 1
+
+/obj/item/clothing/head/celtic/equipped(M as mob, head)
+	if(istype(M, /mob/living/carbon/human))
+		var/mob/living/carbon/human/H = M
+		if(H.head != src)
```

Comment:
No no I mean.

`equipped` passes the slot the object was equipped to, so you can just check whether the second parameter (which you named `head` is `slot_head`).

No need to use `get_item_by_slot` here.